integration/spark/shared/src/main/java/io/openlineage/spark/agent/util/PathUtils.java

```diff
@@ -171,12 +178,27 @@ private static Optional<String> getGlueArn(
 
     Optional<String> accountId =
         SparkConfUtils.findSparkConfigKey(sparkConf, "spark.glue.accountId");
-
-    if (!region.isPresent() || !accountId.isPresent()) {
+    // For AWS Glue catalog in EMR spark
+    // Glue catalog with EMR guide:
+    // https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-glue.html
+    Optional<String> glueCatalogIdForEMR =
+        SparkConfUtils.findSparkConfigKey(sparkConf, "hive.metastore.glue.catalogid");
+    // For AWS Glue access in Athena for Spark
+    // Guide: https://docs.aws.amazon.com/athena/latest/ug/spark-notebooks-cross-account-glue.html
+    Optional<String> glueCatalogIdForAthena =
+        SparkConfUtils.findSparkConfigKey(sparkConf, "spark.hadoop.hive.metastore.glue.catalogid");
```

Comment:
There should be findHadoopConfigKey

Reply:
So these configs can passed to spark configs as well. 
This is from the AWS docs guide.

Reply:
All configs with `spark.hadoop.` prefix are passed to hadoopConf by Spark (with prefix being stripped).
Also the same config could be passed via `hive-site.xml`, and ij this case it will not be available in SparkConf at all.

Reply:
Ackd.  Lemme test this and fix the code

Reply:
Fixed with changes to read from Hadoop config.  You were right and SparkHadoopUtil was doing this copy.